lib/styled.js

```diff
@@ -0,0 +1,82 @@
+'use strict';
+
+let cli   = require('..');
+
+String.prototype.toTitleCase = function() {
```

Comment:
this breaks the linter, and it should. In javascript extending `String` can result in very bad things. Namely, if `heroku-cli-util` is required in 2 different places there can be incompatibilities.

This isn't ruby :p